unpackaged/config/con_merge_testing/classes/REG_ContactMergeUniqueField_TEST.cls

```diff
@@ -0,0 +1,53 @@
+@isTest
+public class REG_ContactMergeUniqueField_TEST {
```

Comment:
I use `REG__` instead of `CON_` as prefix on the test so in the future, regression flow can query other tests to run

Reply:
Works for me! Also, we can easily change these later since they're unpackaged.